static/app/views/alerts/rules/metric/ruleForm.tsx

```diff
@@ -861,96 +860,86 @@ class RuleFormContainer extends AsyncComponent<Props, State> {
       />
     );
 
+    const hasAlertWrite = hasEveryAccess(['alerts:write'], {organization, project});
```

Comment:
This is the only change on this file. We'll look for `alerts:write` on `project.access`, so team-admins can save on the form too.